docs/java-your-first-app.md

```diff
@@ -0,0 +1,163 @@
+---
+id: java-run-your-first-app
+title: Run your first Temporal application with the Java SDK
+sidebar_label: Run your first app
```

Comment:
is this intended to replace "Quick Start"? am a little unclear where this doc sits vs "Quick Start" vs "Get Started" vs "How To".

(suggestion ust for the sake of inciting discussion, pls feel free to ignore) - how about "5 Minute Tutorial"?

Reply:
posted to Slack to start discussion